testing/liberasurecode/APKBUILD

```diff
@@ -0,0 +1,36 @@
+# Contributor: Ehud Kaldor <ehud@unfairfunction.org>
+# Maintainer: Ehud Kaldor <ehud@unfairfunction.org>
+pkgname=liberasurecode
+pkgver=1.5.0
+pkgrel=0
+pkgdesc="Erasure Code API library written in C with pluggable Erasure Code backends"
+url="https://github.com/openstack/liberasurecode"
+arch="all"
+license="COPYING" 
```

Comment:
I don’t know any license named COPYING and it’s definitely not any SPDX license...